src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_IIfStatement.vb

```diff
@@ -0,0 +1,143 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports Microsoft.CodeAnalysis.Semantics
+Imports Microsoft.CodeAnalysis.Test.Utilities
+Imports Microsoft.CodeAnalysis.VisualBasic.Syntax
+Imports Roslyn.Test.Utilities
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.UnitTests.Semantics
+
+    Partial Public Class IOperationTests
+        Inherits SemanticModelTestBase
+
+        <Fact(), WorkItem(17601, "https://github.com/dotnet/roslyn/issues/17601")>
```

Comment:
Can we also port the C# tests where you have a constant expression for the if condition?